### PR TITLE
Translated readme.

### PR DESCRIPTION
**Pull request by davidhsrtoyota**
Translation complete.
Please review.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,17 +1,17 @@
 Overview
 +++++++++++++++++++++
 
-- HSR-Bのコントローラ・センサなどの実機・Gazebo共通設定ファイル管理パッケージを管理する。
+- Manages the common configuration files management package for the HSR-B controllers, sensors, and such for the actual robot and Gazebo.
```
Comment (yuka-hashiguchi) [inline README.rst:4]: I think the Japanese explanation looks wordy :(
How about 
```
 Manages the common configuration files for the HSR-B controllers, sensors, and such for the actual robot and Gazebo.
```

**Review by yuka-hashiguchi — COMMENTED** [review]
Thank you for translating! Let me put a comment.
(It's not your problem but the original sentence)

File: README.rst (hunk 1 above)
Comment (davidhsrtoyota) [reply README.rst:4]: Ah, yes.  I see what you did there.
This looks good.
I will edit the text and re-push it.

**Review by yuka-hashiguchi — APPROVED** [review]
LGTM

**Pull request merged**